userguide/content/en/docs/Adding content/lookandfeel.md

```diff
@@ -191,6 +191,74 @@ See the [Mermaid documentation](https://mermaid-js.github.io/mermaid/#/Setup?id=
 
 Settings can also be overridden on a per-diagram basis by making use of the `%%init%%` header at the start of the diagram definition.  See the [Mermaid theming documentation](https://mermaid-js.github.io/mermaid/#/theming?id=themes-at-the-local-or-current-level).
 
+## UML Diagrams with Plantuml
+
+[plantuml](https://plantuml.com/en/) is a component that allows to quickly write : Sequence diagram, Usecase diagram
+Class diagram, Object diagram, Activity diagram (here is the legacy syntax), Component diagram, Deployment diagram, State diagram, Timing diagram.
+
+Diagrams are defined using a simple and intuitive language. ([see PlantUML Language Reference Guide](https://plantuml.com/en/guide)). As is the case with mermaid, plantuml support in docsy provide the added advantage of editting UML diagrams using text only.
+
+The following example shows a use case diagram:
+
+````
+```plantuml
+participant participant as Foo
+actor       actor       as Foo1
+boundary    boundary    as Foo2
+control     control     as Foo3
+entity      entity      as Foo4
+database    database    as Foo5
+collections collections as Foo6
+queue       queue       as Foo7
+Foo -> Foo1 : To actor 
+Foo -> Foo2 : To boundary
+Foo -> Foo3 : To control
+Foo -> Foo4 : To entity
+Foo -> Foo5 : To database
+Foo -> Foo6 : To collections
+Foo -> Foo7: To queue
+```
+````
+
+Automatically renders to:
+
+```plantuml
+participant participant as Foo
+actor       actor       as Foo1
+boundary    boundary    as Foo2
+control     control     as Foo3
+entity      entity      as Foo4
+database    database    as Foo5
+collections collections as Foo6
+queue       queue       as Foo7
+Foo -> Foo1 : To actor 
+Foo -> Foo2 : To boundary
+Foo -> Foo3 : To control
+Foo -> Foo4 : To entity
+Foo -> Foo5 : To database
+Foo -> Foo6 : To collections
+Foo -> Foo7: To queue
+```
```

Comment:
This isn't actually rendering in the Netlify preview, can you check what's going on?

Reply:
Probably related to the javascript not loading or something around that. Do you have a link to take a look?

Reply:
It's here: https://deploy-preview-467--docsydocs.netlify.app/docs/adding-content/lookandfeel/#uml-diagrams-with-plantuml

If you look at the checks section on the PR page here in GitHub, one of the checks is "deploy preview", and if you click **Details** for that you get a preview version of the Docsy user guide with your PR applied. It's a very handy Netlify feature for checking changes.

Reply:
Should be working now. It was not enables in the config.toml file